koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -1014,11 +1015,20 @@ def handler(self, src, target, opts=None):
             if koji_build_id:
                 all_koji_builds.append(koji_build_id)
 
-        return {
+            user_warnings = result.get('user_warnings')
```

Comment:
Do I see correctly that this whole cycle can do 0 or 1 iteration? Something I call `if`? but contains lot of code around to make it just harder?